docs/src/pages/components/masonry/BasicMasonry.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import Masonry from '@material-ui/lab/Masonry';
+import MasonryItem from '@material-ui/lab/MasonryItem';
+
+export default function BasicMasonry() {
+  return (
+    <Masonry columns={4} spacing={1}>
+      {heights.map((height, index) => (
+        <MasonryItem key={index}>
+          <Box
+            sx={{
+              textAlign: 'center',
+              height,
```

Comment:
```suggestion
              height: '100%',
```

Reply:
I think this should still be `height` so that each `Box` has its own height.

Reply:
I propose 100% here to demonstrate that the parent of the Box has the correct height.